src/toil/worker.py

```diff
@@ -133,9 +134,7 @@ def badWorker():
 
     setLogLevel(config.logLevel)
 
-    tempRootDir = config.workDir
-    if tempRootDir is not None and not os.path.exists(tempRootDir):
-        raise RuntimeError("The temporary directory specified by workDir: %s does not exist" % tempRootDir)
+    tempRootDir = getToilWorkDir(config.workDir)
```

Comment:
Pass entire config. Extracting a specific attribute is an implementation detail of `getToilWorkDir()`